src/main/java/edu/greenblitz/robotName/subsystems/swerve/Chassis/ChassisConstants.java

```diff
@@ -0,0 +1,27 @@
+package edu.greenblitz.robotName.subsystems.swerve.Chassis;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+
+public class ChassisConstants {
+
+    static final Pose2d INITIAL_ROBOT_POSITION = new Pose2d(0, 0, new Rotation2d(0));
+    public static final double FRONT_LEFT_X_POSITION = 0.3020647;
+    public static final double FRONT_LEFT_Y_POSITION = 0.25265;
+    public static final Translation2d[] SWERVE_LOCATIONS_IN_SWERVE_KINEMATICS_COORDINATES = new Translation2d[]{
+            new Translation2d(FRONT_LEFT_X_POSITION, FRONT_LEFT_Y_POSITION), /*fl*/
```

Comment:
It's like we could have used another const instead of those comments 🙃
Also, I'm not sure why we would need to use FL Studio, I can ask Itay if it is useful for them.

![image](https://github.com/GreenBlitz/Robot-Template/assets/26098399/4d42d8bf-ce9b-4f33-9db6-3d80ba540813)

Reply:
(The logo of FL Studio)

Reply:
ye i know wot is FL studio